public/resources/remix/index.js

```diff
@@ -6,122 +6,126 @@
-      detailsBar.addClass("collapsed");
-      return false;
-    });
-
-    $(".mouse-mode").on("mouseenter",function(){
-      detailsBar.removeClass("collapsed");
-    });
-
-    $(".mouse-mode").on("mouseleave",function(){
-      detailsBar.addClass("collapsed");
-    });
   }
 
-  function injectDetailsBar($, metadata, callback) {
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.responseType = "text";
+    var url = metadata.host + "/projects/remix-bar";
+    var data = {
+      host: metadata.host,
+      id: metadata.projectId,
+      title: metadata.projectTitle,
+      author: metadata.projectAuthor,
+      updated: metadata.dateUpdated
+    };
+
+    xmlhttp.onreadystatechange = function() {
+      if (xmlhttp.readyState === XMLHttpRequest.DONE ) {
+        var response = xmlhttp.response;
+        var textStatus = xmlhttp.statusText;
+        if (xmlhttp.status === 200) {
+          var fragmentBody = document.createElement("body");
+          var docFragment = document.createDocumentFragment();
+          docFragment.appendChild(fragmentBody);
+          fragmentBody.innerHTML = response;
+          var remixBar = fragmentBody.querySelector("div");
+          document.body.appendChild(remixBar);
+          callback(null);
+        } else if (xmlhttp.status >= 400) {
+          callback(textStatus);
+        }
       }
-    })
-    .done(function(response) {
-      $(response).prependTo("body");
-      callback(null, $);
-    })
-    .fail(function(jqXHR, textStatus) {
-      callback(textStatus);
-    });
+    };
+
+    xmlhttp.open("GET", url, true);
+    xmlhttp.send(data);
   }
 
-  function injectStyleSheets($, metadata) {
-    var stylesheets =
-      "<link href=\"" + metadata.host + "/resources/remix/clean-slate.min.css\" rel=\"stylesheet\">\n" +
-      "<link href=\"" + metadata.host + "/resources/remix/style.css\" rel=\"stylesheet\">\n";
-    $("head").append(stylesheets);
+  function injectStyleSheets(metadata) {
+    var stylesheet = document.createElement("link");
+    stylesheet.rel = "stylesheet";
+    stylesheet.href = metadata.host + "/resources/remix/clean-slate.min.css";
+    document.head.appendChild(stylesheet);
+
+    stylesheet = document.createElement("link");
+    stylesheet.rel = "stylesheet";
+    stylesheet.href = metadata.host + "/resources/remix/style.css";
+    document.head.appendChild(stylesheet);
   }
 
-  function getMetadata($) {
+  var hasDataRemixAttribute = function(element) {
+    var name = element.getAttribute("name");
+    if (!name) {
+      return false;
+    }
+    return name.startsWith("data-remix-");
```

Comment:
This way you don't have to do the null check you do above either because `.test(null)` and `.test(undefined)` automatically returns false.